dev/make-distribution.sh

```diff
@@ -42,6 +42,8 @@ function exit_with_usage {
   echo "usage:"
   cl_options="[--name] [--tgz] [--mvn <mvn-command>]"
   echo "make-distribution.sh $cl_options <maven build options>"
+  echo "Decrease build times by adding the -T option: -T 1C will result in Maven using one thread per core."
+  echo "For example, dev/make-distribution.sh -T 1C --name hadoop-2.7.2 -Psparkr -Pyarn -Phadoop-2.7 -Phive -Phive-thriftserver -Dscala-2.11"
```

Comment:
Yeah, that's not a way to specify options to mvn inside the script. What about just `-T 1C -Pyarn...`  If that doesn't work there may not be a way to specify this and you could just back this out.